hoomd/md/pair/aniso.py

```diff
@@ -65,8 +65,7 @@ class Dipole(AnisotropicPair):
 
     Args:
         nlist (`hoomd.md.nlist.NList`): Neighbor list
-        r_cut (float): Default cutoff radius (units: [length]).
-        r_on (float): Default turn-on radius (units: [length]).
+        default_r_cut (float): Default cutoff radius (units: [length]).
```

Comment:
Why are we not changing `r_on` ? And related to this question: @b-butler could you remind me why only `r_cut` is the valid argument shown in the documentation but not `r_on`? (in the doc: `class hoomd.md.pair.aniso.Dipole(nlist, r_cut=None, mode='none')`)

Reply:
`r_on` is not in the constructor, but squeaked through the documentation somehow. This is just removing it. `r_on` does not exist for the anisotropic potentials.